web-app/js/portal/lang/en.js

```diff
@@ -149,6 +149,8 @@ OpenLayers.Lang.en = OpenLayers.Util.extend(OpenLayers.Lang.en, {
     btnSave: 'Save',
     btnCancel: 'Cancel',
 
+    errorLoadingCollection: 'Error loading collection with UUID \'{0}\'',
```

Comment:
@kereid @pblain If any of you could assist me with the wording in the message box, it'd be great. Current message is succinct and might not be very user friendly.